testing/environments/docker/kibana/Dockerfile-snapshot

```diff
@@ -2,7 +2,7 @@ FROM debian:jessie
 
 ENV VERSION 5.0.0
 ENV FILENAME_VERSION 5.0.0-linux-x86_64
-ENV URL http://staging.elastic.co/5.0.0-731e78df/downloads/kibana/kibana-${FILENAME_VERSION}.tar.gz
```

Comment:
Yeah, I didn't try it, but I can in a future PR. I imagine it should be somehow possible.